shell/platform/darwin/ios/framework/Source/FlutterSemanticsScrollView.h

```diff
@@ -20,6 +20,8 @@ NS_ASSUME_NONNULL_BEGIN
  */
 @interface FlutterSemanticsScrollView : UIScrollView
 
+@property(nonatomic, assign) SemanticsObject* _Nullable semanticsObject;
```

Comment:
```suggestion
@property(nonatomic, assign, nullable) SemanticsObject* semanticsObject;
```

Reply:
Live footage of me writing ObjC code:
https://memegenerator.net/img/instances/72953023/i-have-no-idea-what-im-doing.jpg